ansible/roles/ocp-workload-edge-deployments/tasks/remove_workload.yml

```diff
@@ -1,6 +1,44 @@
 ---
 # Implement your Workload removal tasks here
+- name: Cleaning up the environment
+  debug:
+    msg: "Cleaning up the environment.."
+
+- name: Delete project IoT Development
+  shell: "oc delete project iot-development"
+  ignore_errors: true
```

Comment:
What is this workload target ?
is it a shared cluster or is it a cluster that will be also destroyed ?

If it's a shared-cluster, then please do not use `ignore_errors` here